src/CMakeLists.txt

```diff
@@ -80,15 +80,18 @@ if (MSVC)
         # string(APPEND CMAKE_EXE_LINKER_FLAGS_${build_type} " /LTCG:INCREMENTAL /OPT:REF")
     endforeach()
     
-    add_compile_options(/await)
     add_compile_options(/permissive-)
-    string(APPEND CMAKE_CXX_FLAGS_DEBUG " /JMC")
+
+    if (CMAKE_CXX_COMPILER MATCHES "clang-cl")
+        add_compile_options(-Wno-delete-non-virtual-dtor -mcx16 -fno-delayed-template-parsing -Xclang -fcoroutines-ts)
+    else()
+        add_compile_options(/await)
+    endif()
 
     # Explicitly configure _DEBUG preprocessor macro 
     string(APPEND CMAKE_CXX_FLAGS_DEBUG " /D_DEBUG")
 else()
     add_compile_options(-stdlib=libc++)
-    add_compile_options(-fcoroutines-ts)
```

Comment:
Why did you remove coro support for linux but leave it in for windows?

Reply:
Testing with non-Windows now...